sdk/servicebus/service-bus/test/internal/peekMessagesOptions.spec.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Should this be under public instead of under internal?

Reply:
I intentionally put it under internal to reduce the discoverability of it.

Reply:
I think the tests in the public folder is meant for tests that can be run in the min/max testing pipelines and I think this test is a good candidate for that.

Reply:
fair. I moved it to public